api/metrics/src/main/java/io/opentelemetry/api/metrics/ObservableMeasurement.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.metrics;
+
+/**
+ * A mechanism for observing measurments.
+ *
+ * <p>see {@link ObservableDoubleMeasurement} or {@link ObservableLongMeasurement}.
+ */
+public interface ObservableMeasurement {}
```

Comment:
with no methods and no types does this have enough value to keep?

Reply:
It helped in the SDK implementation.   I can remove it, but you don't see the code which uses this yet.